### ISSUE TITLE
[FEATURE]: Make a landing page

### ISSUE THREAD

**Issue by DugarRishab** [DugarRishab/Algo.io#66]
## Description

Make a stunning Landing Page using react for Algo.io. The landing page must be of high quality and unique design. You may use MUI, but the design must be unique. You are free to choose your color schemes but try to make it in dark-mode. The landing page should be of at-most 1 page. 

## Requirements

- [ ] Use React
- [ ] The design must be responsive
- [ ] Make the following directories inside `/client/src/` : 
  - [ ] `/components/` to store all components. Note: Each component should be in their respective folders inside '/compnents/`
  - [ ] `/pages/` to store all pages
- [ ] Route all the pages using `routes.js` file inside `/client/src/`. 
- [ ] Create a `template.md` file in `/client/src/` and write details about the color schema for future developers.
- [ ] Provide the full-screen screenshot of the final rendered page. 

## Note: 

- In the PR, please disregard the template because it was created for other purpose. Please copy the Requirements section from this description to your PRs description and mark all that you have done. 
- Please note that this issue will not be assigned on a **first come first serve** basis, rather on credit basis by judging the repos in your profile.

**Comment by Harikrishna3** [comment]
I'd like to work on this; Would you kindly assign it to me?

**Comment by Naveen-Nair** [comment]
@DugarRishab I'd love to contribute to your project! Would you assign it to me?

**Comment by DugarRishab** [comment]
@Harikrishna3 I am assigning this to you. Go ahead with your PR.

**Comment by DugarRishab** [comment]
any updates @Harikrishna3 ?

**Comment by Harikrishna3** [comment]
I am still designing the page.

**Comment by Harikrishna3** [comment]
would you plz guide me.

**Comment by amitkroutthedev** [comment]
I'd like to work on this; Would you kindly assign it to me?

**Comment by DugarRishab** [comment]
> would you plz guide me.

Yeah, so you can make a menu on the left, and on the right, the algorithm/code will be shown. There will be a navbar and a footer. That's  it.

**Comment by Harikrishna3** [comment]
Ok

**Comment by Harikrishna3** [comment]
![image](https://user-images.githubusercontent.com/86600101/195382525-a8e8ce27-c0c9-4334-bfe8-6319cc8f6500.png)

**Comment by Harikrishna3** [comment]
check this.

**Comment by Harikrishna3** [comment]
can I PR it

**Comment by Harikrishna3** [comment]
![image](https://user-images.githubusercontent.com/86600101/195385770-a82f747b-b7ca-497b-9bd6-b376834fcae8.png)

**Comment by Harikrishna3** [comment]
Hey its response too.

**Comment by Harikrishna3** [comment]
![image](https://user-images.githubusercontent.com/86600101/195394224-43fab8d3-45d2-4b23-9fc1-05a97a019120.png)
![image](https://user-images.githubusercontent.com/86600101/195394244-0ac94742-2252-4c04-934c-617b2639694c.png)

**Comment by DugarRishab** [comment]
Yeah, LGTM. Change the font of the MAIN tab. Otherwise everything looks amazing. And can u move the tabs to the right.

**Comment by Harikrishna3** [comment]
![image](https://user-images.githubusercontent.com/86600101/195643463-b60f6697-9efe-4a52-a411-a981038fbf31.png)

**Comment by Harikrishna3** [comment]
![image](https://user-images.githubusercontent.com/86600101/195644901-a9ad2870-0f41-4a61-9fa2-9c0ccec52f24.png)

**Comment by Harikrishna3** [comment]
which one is ok

**Comment by DugarRishab** [comment]
> ![image](https://user-images.githubusercontent.com/86600101/195643463-b60f6697-9efe-4a52-a411-a981038fbf31.png)

this on looks awesome

**Comment by DugarRishab** [comment]
send a PR when you @Harikrishna3 can. Remember to do everything mentioned in the description.

**Comment by garg-Megha** [comment]
@DugarRishab I'd like to contribute to this; Would you kindly assign it to me?

**Comment by DugarRishab** [comment]
> @DugarRishab I'd like to contribute to this; Would you kindly assign it to me?

Hey @garg-Megha, I looked at your GitHub profile and I did not find any relevant repo. But go ahead and make the design and send a ss, if it looks good, I will assign it to you.

**Comment by vaibhavshah03** [comment]
I'd like to contribute to this; Would you kindly assign it to me?
